packages/components/scripts/generateDSRComponents.ts

```diff
@@ -213,13 +214,29 @@ import { get${componentName}Css } from '${stylesBundleImportPath}';
         .replace(/return this\.selectRef\.selectedIndex;/, 'return 0;') // select-wrapper-dropdown
         .replace(/determineDirection\(this\.props\)/, "'down'") // select-wrapper-dropdown
         .replace(/(this\.)props\.(isDisabledOrLoading)/g, '$1$2') // button, button-pure
-        .replace(/(const (?:iconProps|btnProps|linkProps)) =/, '$1: any ='); // workaround typing issue
+        .replace(/(const (?:iconProps|btnProps|linkProps)) =/, '$1: any =') // workaround typing issue
```

Comment:
It would be nice to have unit tests for `generateDSRComponents` in order to:
1) know that all the cases are covered and it works correctly in all the cases
2) better / easier understand, what is does

Reply:
For sure, currently only the produces output is tested. The code producing the output is not.